data/hai/hai reveal 1 intro.txt

```diff
@@ -778,10 +778,18 @@ mission "Hai Leaks Response 4"
 					accept
 				`	"I'll beat the Lich, and do my best to capture the Geocoris and bring them safely home."`
 					accept
+	# Escorts manually defined to have weapons comparatively suitable for disabling, as would be appropriate in a mission where the *hopeful* outcome is a rescue.
```

Comment:
These ships aren't going to attack the one we want to take in tact anyway.

Reply:
No, but it's been raised as strange that they deploy tracker-heavy fleets against foes who are potentially carrying captured Hai, somewhat regardless of whether they mechanically target the Geocoris.